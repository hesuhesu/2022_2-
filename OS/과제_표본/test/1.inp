200
52277 Everett Gilpin
11369 Genna Bussell
13707 Lavona Hommel
43176 Lee Baize
23695 Polly Tolleson
09154 Jermaine Baize
16565 Delores Burson
35627 Asley Baize
44217 Darlena Aycock
06729 Vikki Batres
41621 Elmo Batres
52224 Charleen Bussell
22987 Jodi Kester
12733 Elois Burson
15455 Teresita Aycock
04204 Cherie Aycock
01869 Ashley Briscoe
22253 Dina Styers
13790 Adah Baize
02634 Rodger Aycock
02143 Xiomara Goettl
38961 Wonda Traver
37450 Franchesca Aycock
24792 Chloe Mahmood
46952 Hertha Pearman
35309 Haley Schlenker
30098 Tricia Hommel
43594 Mackenzie Rollman
30525 Lynsey Gilpin
02416 Julene Goettl
37020 Margo Gilpin
03367 Everett Hommel
34471 Tristan Baize
08210 Jolanda Plain
37797 Rose Stpierre
15731 Hannelore Burson
44895 Charmain Goettl
03804 Benny Dimattia
42852 Glinda Burson
26459 Amie Sneed
21318 Shonna Baize
17626 Yer Eyer
27053 Mari Aycock
34104 Brandee Batres
11755 Shena Livengood
10431 Geneva Burson
28090 Toni Fausto
23081 Shon Mickles
21409 Corina Baize
38431 Estell Goettl
29891 Elane Twiggs
19126 Shila Gilpin
47083 Breanne Mule
45309 Candelaria Poore
48599 Cordia Hommel
26466 Tasia Hagemeier
46228 Aracely Saiz
07565 Marcie Hommel
12240 Keira Walburn
04960 Bobbi Doyle
16897 Travis Rollman
05547 Sherry Hommel
35072 Danica Rollman
31863 Lavonda Okafor
08815 Marvis Hommel
51075 Christinia Aycock
06793 Coy Batres
46045 Loree Rollman
39775 Vinnie Hommel
51979 Tiesha Burson
48131 Kristofer Hommel
23075 Hannah Kriger
36776 Shaunda Burson
13373 Werner Shadle
46625 Tori Rollman
07883 Jesenia Burson
18127 Sherrie Burson
43008 Rona Rollman
12332 Harmony Baize
50288 Dominque Aycock
52875 Diana Gilpin
52214 Barbar Goettl
53214 Bettie Rollman
24345 Jenny Baize
27342 Jen Batres
51303 Danille Hambright
32986 Willodean Hadsell
16860 Ludivina Fane
35986 Miss Gilpin
41249 Kasie Baize
42904 Sandee Harbaugh
18655 Arletta Rollman
26515 Jarred Baize
32813 Alda Luton
22530 China Taul
04583 Marlo Bussell
07092 Lynwood Liston
39540 Rana Burson
19927 Kirstie Hagemeier
20275 Karri Bussell
28938 Mignon Hommel
33637 Eleanore Burson
37428 Kraig Aycock
05026 Sonia Mctaggart
29119 Joeann Womer
49217 Mara Tabor
36205 Deloise Bussell
49736 Taylor Goettl
08995 Mariana Gartman
45520 Robby Call
02906 Charlesetta Aycock
31373 Rosana Goettl
11933 Latasha Goettl
37972 Tiffany Hommel
01328 Cristina Gibney
48264 Odilia Goettl
33358 Bryant Baize
49995 Shaniqua Goettl
42529 Ignacio Bussell
21084 Eufemia Hommel
34194 Verline Batres
15001 Elin Baize
23865 Lesha Rollman
34920 Jacki Baize
25652 Rikki Wampler
21712 Grisel Pineiro
08265 Ezra Dibiase
04292 Victor Blade
04658 Amalia Tillery
16012 Lenore Purdum
40090 Junita Aycock
09455 Thersa Bussell
17399 August Aycock
05355 Sherika Bussell
40460 Gretchen Pflum
44426 Mathew Gilpin
17035 Tatyana Mcglynn
08486 Kirstie Baize
06523 Troy Reichert
50670 Mabelle Goettl
00922 Palma Rollman
11247 Larhonda Bussell
36670 Kimberlie Gilpin
42115 Betsey Goettl
51747 Cristin Gilpin
32062 Magaret Chenail
26138 Mui Rollman
09713 Melaine Pinckney
12055 Sadie Baize
48728 Jaquelyn Gilpin
46778 Hallie Pendergast
18380 Miguel Dorman
25385 Cody Bussell
09957 Corinna Baize
29083 Domenic Batres
33645 Bert Condron
49447 Suzi Aycock
08115 Denae Corbeil
29220 Gabrielle Delisle
08351 Ali Goettl
45946 Marjorie Hommel
47457 Faviola Burson
06753 Nada Ruge
16203 Collette Grauer
32404 Rigoberto Bruton
39066 Odette Gilpin
18094 Danika Burson
00704 Todd Aycock
14544 Elza Gilpin
06388 Sudie Gilpin
18844 Tammera Gilpin
52390 Jane Burson
27790 Richie Aycock
14203 Kala Churchill
06020 Mindy Aycock
16810 Arianne Goettl
10805 Kristal Illingworth
44178 Brady Dougherty
01639 Faustino Burson
20678 Jean Goettl
23477 Providencia Batres
28165 Amberly Mealey
15585 Leann Brunet
26578 Shanita Aycock
12895 Sylvia Goettl
12822 Tonja Gilpin
26531 Karyn Lute
40944 Allegra Shaddix
19497 Janette Gilpin
24997 Fabiola Cavin
36324 Kum Baize
38848 Jaimee Crespo
29593 Tobie Correa
49940 Exie Warshaw
30994 Doyle Batres
47902 Rosaura Amodeo
41162 Jude Gilpin
28647 Rosenda Batres
44000 Dagny Bencomo
22062 Russell Hommel
