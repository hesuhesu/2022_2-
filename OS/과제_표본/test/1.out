00704 Todd        Aycock
00922 Palma       Rollman
01328 Cristina    Gibney
01639 Faustino    Burson
01869 Ashley      Briscoe
02143 Xiomara     Goettl
02416 Julene      Goettl
02634 Rodger      Aycock
02906 Charlesetta Aycock
03367 Everett     Hommel
03804 Benny       Dimattia
04204 Cherie      Aycock
04292 Victor      Blade
04583 Marlo       Bussell
04658 Amalia      Tillery
04960 Bobbi       Doyle
05026 Sonia       Mctaggart
05355 Sherika     Bussell
05547 Sherry      Hommel
06020 Mindy       Aycock
06388 Sudie       Gilpin
06523 Troy        Reichert
06729 Vikki       Batres
06753 Nada        Ruge
06793 Coy         Batres
07092 Lynwood     Liston
07565 Marcie      Hommel
07883 Jesenia     Burson
08115 Denae       Corbeil
08210 Jolanda     Plain
08265 Ezra        Dibiase
08351 Ali         Goettl
08486 Kirstie     Baize
08815 Marvis      Hommel
08995 Mariana     Gartman
09154 Jermaine    Baize
09455 Thersa      Bussell
09713 Melaine     Pinckney
09957 Corinna     Baize
10431 Geneva      Burson
10805 Kristal     Illingworth
11247 Larhonda    Bussell
11369 Genna       Bussell
11755 Shena       Livengood
11933 Latasha     Goettl
12055 Sadie       Baize
12240 Keira       Walburn
12332 Harmony     Baize
12733 Elois       Burson
12822 Tonja       Gilpin
12895 Sylvia      Goettl
13373 Werner      Shadle
13707 Lavona      Hommel
13790 Adah        Baize
14203 Kala        Churchill
14544 Elza        Gilpin
15001 Elin        Baize
15455 Teresita    Aycock
15585 Leann       Brunet
15731 Hannelore   Burson
16012 Lenore      Purdum
16203 Collette    Grauer
16565 Delores     Burson
16810 Arianne     Goettl
16860 Ludivina    Fane
16897 Travis      Rollman
17035 Tatyana     Mcglynn
17399 August      Aycock
17626 Yer         Eyer
18094 Danika      Burson
18127 Sherrie     Burson
18380 Miguel      Dorman
18655 Arletta     Rollman
18844 Tammera     Gilpin
19126 Shila       Gilpin
19497 Janette     Gilpin
19927 Kirstie     Hagemeier
20275 Karri       Bussell
20678 Jean        Goettl
21084 Eufemia     Hommel
21318 Shonna      Baize
21409 Corina      Baize
21712 Grisel      Pineiro
22062 Russell     Hommel
22253 Dina        Styers
22530 China       Taul
22987 Jodi        Kester
23075 Hannah      Kriger
23081 Shon        Mickles
23477 Providencia Batres
23695 Polly       Tolleson
23865 Lesha       Rollman
24345 Jenny       Baize
24792 Chloe       Mahmood
24997 Fabiola     Cavin
25385 Cody        Bussell
25652 Rikki       Wampler
26138 Mui         Rollman
26459 Amie        Sneed
26466 Tasia       Hagemeier
26515 Jarred      Baize
26531 Karyn       Lute
26578 Shanita     Aycock
27053 Mari        Aycock
27342 Jen         Batres
27790 Richie      Aycock
28090 Toni        Fausto
28165 Amberly     Mealey
28647 Rosenda     Batres
28938 Mignon      Hommel
29083 Domenic     Batres
29119 Joeann      Womer
29220 Gabrielle   Delisle
29593 Tobie       Correa
29891 Elane       Twiggs
30098 Tricia      Hommel
30525 Lynsey      Gilpin
30994 Doyle       Batres
31373 Rosana      Goettl
31863 Lavonda     Okafor
32062 Magaret     Chenail
32404 Rigoberto   Bruton
32813 Alda        Luton
32986 Willodean   Hadsell
33358 Bryant      Baize
33637 Eleanore    Burson
33645 Bert        Condron
34104 Brandee     Batres
34194 Verline     Batres
34471 Tristan     Baize
34920 Jacki       Baize
35072 Danica      Rollman
35309 Haley       Schlenker
35627 Asley       Baize
35986 Miss        Gilpin
36205 Deloise     Bussell
36324 Kum         Baize
36670 Kimberlie   Gilpin
36776 Shaunda     Burson
37020 Margo       Gilpin
37428 Kraig       Aycock
37450 Franchesca  Aycock
37797 Rose        Stpierre
37972 Tiffany     Hommel
38431 Estell      Goettl
38848 Jaimee      Crespo
38961 Wonda       Traver
39066 Odette      Gilpin
39540 Rana        Burson
39775 Vinnie      Hommel
40090 Junita      Aycock
40460 Gretchen    Pflum
40944 Allegra     Shaddix
41162 Jude        Gilpin
41249 Kasie       Baize
41621 Elmo        Batres
42115 Betsey      Goettl
42529 Ignacio     Bussell
42852 Glinda      Burson
42904 Sandee      Harbaugh
43008 Rona        Rollman
43176 Lee         Baize
43594 Mackenzie   Rollman
44000 Dagny       Bencomo
44178 Brady       Dougherty
44217 Darlena     Aycock
44426 Mathew      Gilpin
44895 Charmain    Goettl
45309 Candelaria  Poore
45520 Robby       Call
45946 Marjorie    Hommel
46045 Loree       Rollman
46228 Aracely     Saiz
46625 Tori        Rollman
46778 Hallie      Pendergast
46952 Hertha      Pearman
47083 Breanne     Mule
47457 Faviola     Burson
47902 Rosaura     Amodeo
48131 Kristofer   Hommel
48264 Odilia      Goettl
48599 Cordia      Hommel
48728 Jaquelyn    Gilpin
49217 Mara        Tabor
49447 Suzi        Aycock
49736 Taylor      Goettl
49940 Exie        Warshaw
49995 Shaniqua    Goettl
50288 Dominque    Aycock
50670 Mabelle     Goettl
51075 Christinia  Aycock
51303 Danille     Hambright
51747 Cristin     Gilpin
51979 Tiesha      Burson
52214 Barbar      Goettl
52224 Charleen    Bussell
52277 Everett     Gilpin
52390 Jane        Burson
52875 Diana       Gilpin
53214 Bettie      Rollman

Aycock 17
Baize 18
Batres 10
Burson 15
Bussell 10
Gilpin 17
Goettl 16
Hagemeier 2
Hommel 14
Rollman 11
